### PR TITLE
Made info.gui display on player_created

### PR DESCRIPTION
**Pull request by SimonFlapse**
Removed old welcome message and replaced it with a short reminder on how to change keybinding to enable chat.

Leaving it here to see what people think

#282 discusses the problems of people asking questions available in the info.gui. This is a suggestion on how to help people notice this gui.

### CODE REVIEW

File: control.lua [hunk 1]
```diff
@@ -60,13 +60,16 @@ local function player_created(event)
     end
     player.insert {name = 'iron-gear-wheel', count = 8}
     player.insert {name = 'iron-plate', count = 16}
-    player.print('Welcome to our Server. You can join our Discord at: redmew.com/discord')
-    player.print('Click the question mark in the top left corner for server infomation and map details.')
-    player.print('And remember.. Keep Calm And Spaghetti!')
+
+    player.print('Trouble chatting? Change the keybinding in:')
```
Comment (iltar) [inline control.lua]: People often don't know that the backtick is the key they need to press, perhaps we should also mention this.

Bit of a brainfart, what about a "ping" system? We can parse messages and if we see `@Linaori` and the player is in the map, we could ping that player with a sound.

Comment (SimonFlapse) [reply control.lua]: It's listed in the info GUI already, and you can see the Keybinder inside your keyboard settings. I haven't been able to find the backtick on my keyboard (that Factorio recognize) so I didn't include it.

I like the ping system idea.

**Comment by plague006** [comment]
Conflicts #333

**Comment by SimonFlapse** [comment]
I'll make the necessary changes when #333 has been merged

**Pull request merged**